src/interpolation/compute/compute.test.js

```diff
@@ -54,7 +54,13 @@ test('Default formatter - capitalize two words', () => {
 test('Unknown variable', () => {
   const interpolated = compute({}, 'user.name')
 
-  expect(interpolated).toBe('')
+  expect(interpolated).toBe('[[undefined]]')
```

Comment:
In my opinion, an unknown variable should be interpolated as empty string. We don't want "[[undefined]]" to be printed on documents.

This is related to the new "discovery" method that we should implement. This will help to find "unknown/unmatched variables" in the data and maybe throw a warning in those cases.